reader/nordigen/nordigen.go

```diff
@@ -40,12 +40,37 @@ func transactionsToYnabber(account ynabber.Account, t nordigen.AccountTransactio
 			return nil, fmt.Errorf("failed to parse string to time: %w", err)
 		}
 
+		// Find payee
+		payee := ynabber.Payee("")
+		for _, source := range config.YNAB.PayeeSource {
+			if payee == "" {
+				if source == "name" {
+					// Creditor/debtor name can be used as is
+					if v.CreditorName != "" {
+						payee = ynabber.Payee(v.CreditorName)
+					} else if v.DebtorName != "" {
+						payee = ynabber.Payee(v.DebtorName)
+					}
+				} else if source == "unstructured" {
+					// Unstructured data may need some formatting
+					payee = ynabber.Payee(v.RemittanceInformationUnstructured)
+					s, err := payee.Parsed(config.YNAB.PayeeStrip)
+					if err != nil {
+						log.Printf("Failed to parse payee: %s: %s", string(payee), err)
+					}
+					payee = ynabber.Payee(s)
+				} else {
+					log.Panicf("Unrecognized payee data source %s", source)
+				}
+			}
+		}
```

Comment:
I understand that the idea here is to break the linear scale issue of hardcoding all the banks and their field mapping. It sounds good in theory, but I fear that all banks are going to use slightly different mapping and thus in the end we're gonna end up with an almost 1:1 relationship between `PayeeSource` and the number of banks we "support".

I'm tempted to say that we need to give the user complete control over the mapping by using reflection to implement this in a dynamic way. We can still try and guess the proper mapping in the code based on various techniques but an override setting is required IMO.

Reply:
I don't completely understand what you mean. So far, in comments on other PRs, only your initial solution and using creditorName/debtorName have come up. Initially I actually looked at the possibility of mentioning the field names as payee candidates.

However, going through the fields available in transactions, I find it hard to imagine there would be data useful as payee in any other field. So this covers all cases I can now see.

That doesn't mean it would not require a more complex translation for some banks or even specific users. We should then possibly revisit this. Some of them become so usecase specific that it cannot be coded beforehand in a sensible way. Just a field mapping would not cut it - we might require resorting to regexp and/or splitting of data. Or even mapping numbers to a name ...

If you mean on YNAB side there are very few fields and most of them are very straigthforward. Only Payee is somewhat controversial as YNAB does its suggested category mappings based on that.
Even if we get it just close to right one, it will be immensely helpful to user. They may always edit it by hand (and also memo).

I don't know what else to suggest anymore which would make sense i.e. be relatively simple to use and not require extensive code maintenance effort.

Reply:
You're right, let's try this and see how it works out 🎉